f5_openstack_agent/lbaasv2/drivers/bigip/pool_service.py

```diff
@@ -53,8 +56,28 @@ def create_pool(self, service, bigips):
         :param bigips: Array of BigIP class instances to create pool.
         """
         pool = self.service_adapter.get_pool(service)
+        error = None
+
         for bigip in bigips:
-            self.pool_helper.create(bigip, pool)
+            try:
```

Comment:
Would be nice if you could abstract this pattern of "perform operation, catch exceptions, log error, return result" -- which you did in CCCL. Or maybe that's just something we defer until CCCL integration?

Reply:
Thanks Erik.  My experience with this lead me to implementing CCCL the way it is, but it's a stretch to redo that here.